robottelo/constants.py

```diff
@@ -241,7 +241,7 @@
 DEFAULT_CV = "Default Organization View"
 DEFAULT_PTABLE = "Kickstart default"
 DEFAULT_SUBSCRIPTION_NAME = (
-    'Red Hat Enterprise Linux Server Entry Level, Self-support')
+    'Red Hat Enterprise Linux Server, Premium (Physical or Virtual Nodes)')
```

Comment:
That a little bit risky change that will require running some more tests probably